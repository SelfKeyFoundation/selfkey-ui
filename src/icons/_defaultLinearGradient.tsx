import * as React from 'react';

export const DefaultLinearGradient = () => {
    return (
        <defs>
            <linearGradient x1="11.0842631%" y1="68.33943%" x2="36.0705554%" y2="0%" id="dlg">
                <stop stopColor="#09A8BA" offset="0%"></stop>
                <stop stopColor="#0ABBD0" offset="100%"></stop>
            </linearGradient>
        </defs>
    )
};

export default DefaultLinearGradient;
