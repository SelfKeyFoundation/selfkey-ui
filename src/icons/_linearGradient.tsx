import * as React from 'react';

export type LinearGradientProps = any;

export const LinearGradient = ({ color }: LinearGradientProps) => {
	let color1;
	let color2;
	let id;
	switch (color) {
		case 'orange':
			color1 = '#B44000';
			color2 = '#DB7400';
			id = 'orange-linear-gradient';
			break;
		case 'grey':
			color1 = '#93B0C1';
			color2 = '#647A86';
			id = 'grey-linear-gradient';
			break;
		default:
			color1 = '#09A8BA';
			color2 = '#0ABBD0';
			id = 'default-linear-gradient';
	}

	return (
		<defs>
			<linearGradient x1="11.0842631%" y1="68.33943%" x2="36.0705554%" y2="0%" id={id}>
				<stop stopColor={color1} offset="0%"></stop>
				<stop stopColor={color2} offset="100%"></stop>
			</linearGradient>
		</defs>
	);
};

export default LinearGradient;
