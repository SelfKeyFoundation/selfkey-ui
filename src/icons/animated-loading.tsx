import * as React from 'react';
import { WithStyles, withStyles, createStyles } from '@material-ui/core';
import { SvgIcon } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';

const styles = createStyles({
    root: {
       width: '120px !important',
       height: '30px !important',
    },
});

export const AnimatedLoadingIcon = withStyles(styles)((props: SvgIconProps & WithStyles<typeof styles>) => (
    <SvgIcon {...props} viewBox="0 0 120 30">
        <circle cx="15" cy="15" r="15">
            <animate
                attributeName="r"
                from="15"
                to="15"
                begin="0s"
                dur="0.8s"
                values="15;9;15"
                calcMode="linear"
                repeatCount="indefinite"
            />
            <animate
                attributeName="fillOpacity"
                from="1"
                to="1"
                begin="0s"
                dur="0.8s"
                values="1;.5;1"
                calcMode="linear"
                repeatCount="indefinite"
            />
        </circle>
        <circle cx="60" cy="15" r="9" fillOpacity="0.3">
            <animate
                attributeName="r"
                from="9"
                to="9"
                begin="0s"
                dur="0.8s"
                values="9;15;9"
                calcMode="linear"
                repeatCount="indefinite"
            />
            <animate
                attributeName="fillOpacity"
                from="0.5"
                to="0.5"
                begin="0s"
                dur="0.8s"
                values=".5;1;.5"
                calcMode="linear"
                repeatCount="indefinite"
            />
        </circle>
        <circle cx="105" cy="15" r="15">
            <animate
                attributeName="r"
                from="15"
                to="15"
                begin="0s"
                dur="0.8s"
                values="15;9;15"
                calcMode="linear"
                repeatCount="indefinite"
            />
            <animate
                attributeName="fillOpacity"
                from="1"
                to="1"
                begin="0s"
                dur="0.8s"
                values="1;.5;1"
                calcMode="linear"
                repeatCount="indefinite"
            />
        </circle>
    </SvgIcon>
));

export default AnimatedLoadingIcon;
