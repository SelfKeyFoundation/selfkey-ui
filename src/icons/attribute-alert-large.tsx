import * as React from 'react';
import { SvgIcon, WithStyles, withStyles, createStyles } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';

const styles = createStyles({
    root: {
       width: '38px !important',
       height: '44px !important',
    },
});

export const AttributeAlertLargeIcon = withStyles(styles)((props: SvgIconProps & WithStyles<typeof styles>) => (
	<SvgIcon {...props} viewBox="0 0 38 44" version="1.1">
        <g id="ðŸ’»-ID-Dashboard-attribute-large-icon" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
            <g id="410.1-Marketplace-Applications-Statuses" transform="translate(-195.000000, -379.000000)" fill="#E98548">
                <g id="Content" transform="translate(150.000000, 264.000000)">
                    <g id="Collapse-box-1">
                        <g id="Status-box-1" transform="translate(15.000000, 90.000000)">
                            <path d="M67.91,31.27 C67.9698291,31.5087207 68.0000562,31.7538962 68,32 L68,45.3 C67.9841317,53.5919031 63.5062321,61.23337 56.28,65.3 L50.47,68.57 C49.5571638,69.0831103 48.4428362,69.0831103 47.53,68.57 L41.72,65.3 C34.4937679,61.23337 30.0158683,53.5919031 30,45.3 L30,32 C30,31.32 30.27,29.02 33.66,27.32 C36.73,25.78 41.89,25 49,25 C64.09,25 67.32,28.93 67.91,31.27 Z M65,45.27 L65,32 C65,32 64,28 49,28 C33,28 33,32 33,32 L33,45.3 C32.9995752,52.5239266 36.8947728,59.186664 43.19,62.73 L49,66 L54.81,62.7 C61.1052272,59.156664 65.0004248,52.4939266 65,45.27 Z M48.5,59 C47.6715729,59 47,58.3284271 47,57.5 C47,56.6715729 47.6715729,56 48.5,56 C49.3284271,56 50,56.6715729 50,57.5 C50,58.3284271 49.3284271,59 48.5,59 Z M48.5,33.5 C49.3284271,33.5 50,34.1715729 50,35 L50,51 C50,51.8284271 49.3284271,52.5 48.5,52.5 C47.6715729,52.5 47,51.8284271 47,51 L47,35 C47,34.1715729 47.6715729,33.5 48.5,33.5 Z" id="attribute-large-icon"></path>
                        </g>
                    </g>
                </g>
            </g>
        </g>
	</SvgIcon>
));

export default AttributeAlertLargeIcon;
