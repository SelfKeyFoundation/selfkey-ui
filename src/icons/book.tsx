import * as React from 'react';
import injectSheet, { StyleSheet, StyledComponentProps } from 'react-jss';

import { SvgIcon } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';
import DefaultLinearGradient from './_defaultLinearGradient';

const styles: StyleSheet = {
  root: {
    transition: 'all 0.2s ease-out',
    '&:hover': {
        fill: 'white',
    },
  }
};

export const BookIcon = injectSheet(styles)<SvgIconProps>((props: SvgIconProps & StyledComponentProps)=> (
  <SvgIcon style={{ width: props.width || '68px', height: props.height || '55px', fill: props.fill || 'url(#dlg)' }} {...props} viewBox='0 0 68 55'>
    <DefaultLinearGradient />
    <g id="ðŸ’»-ID-Dashboard" stroke="none" strokeWidth="1" fill="inherit" fillRule="evenodd">
        <g id="403---Identity-Summary" transform="translate(-240.000000, -958.000000)" fill="inherit" fillRule="nonzero">
            <g id="Area-box-2" transform="translate(150.000000, 462.000000)">
                <g id="Additional" transform="translate(14.000000, 390.000000)">
                    <g id="Explanatory" transform="translate(10.000000, 106.000000)">
                        <path d="M71,0 L71,3 L66,3 L66,52 L95.505859,52 C96.284666,53.745175 97.971465,55 100,55 C102.028851,55 103.715042,53.74513 104.494141,52 L134,52 L134,3 L129,3 L129,0 L103,0 C101.789062,0 100.735556,0.576246 100,1.435547 C99.264444,0.576246 98.210938,0 97,0 L71,0 Z M73,2 L97,2 C98.116666,2 99,2.883334 99,4 L101,4 C101,2.883334 101.883334,2 103,2 L127,2 L127,45 L103,45 C101.789062,45 100.735556,45.576246 100,46.435547 C99.264444,45.576246 98.210938,45 97,45 L73,45 L73,2 Z M68,5 L71,5 L71,47 L97,47 C98.116666,47 99,47.883334 99,49 L101,49 C101,47.883334 101.883334,47 103,47 L129,47 L129,5 L132,5 L132,50 L103.085938,50 L102.890625,50.748047 C102.554647,52.042353 101.404427,53 100,53 C98.595573,53 97.444396,52.043513 97.109375,50.75 L96.914062,50 L68,50 L68,5 Z M100,6 C99.4477153,6 99,6.44771525 99,7 C99,7.55228475 99.4477153,8 100,8 C100.552285,8 101,7.55228475 101,7 C101,6.44771525 100.552285,6 100,6 Z M78,10 L78,12 L86,12 L86,10 L78,10 Z M89,10 C88.4477153,10 88,10.4477153 88,11 C88,11.5522847 88.4477153,12 89,12 C89.5522847,12 90,11.5522847 90,11 C90,10.4477153 89.5522847,10 89,10 Z M93,10 C92.4477153,10 92,10.4477153 92,11 C92,11.5522847 92.4477153,12 93,12 C93.5522847,12 94,11.5522847 94,11 C94,10.4477153 93.5522847,10 93,10 Z M100,10 C99.4477153,10 99,10.4477153 99,11 C99,11.5522847 99.4477153,12 100,12 C100.552285,12 101,11.5522847 101,11 C101,10.4477153 100.552285,10 100,10 Z M106,10 L106,12 L114,12 L114,10 L106,10 Z M117,10 C116.447715,10 116,10.4477153 116,11 C116,11.5522847 116.447715,12 117,12 C117.552285,12 118,11.5522847 118,11 C118,10.4477153 117.552285,10 117,10 Z M121,10 C120.447715,10 120,10.4477153 120,11 C120,11.5522847 120.447715,12 121,12 C121.552285,12 122,11.5522847 122,11 C122,10.4477153 121.552285,10 121,10 Z M100,14 C99.4477153,14 99,14.4477153 99,15 C99,15.5522847 99.4477153,16 100,16 C100.552285,16 101,15.5522847 101,15 C101,14.4477153 100.552285,14 100,14 Z M78,16 L78,18 L86,18 L86,16 L78,16 Z M89,16 C88.4477153,16 88,16.4477153 88,17 C88,17.5522847 88.4477153,18 89,18 C89.5522847,18 90,17.5522847 90,17 C90,16.4477153 89.5522847,16 89,16 Z M93,16 C92.4477153,16 92,16.4477153 92,17 C92,17.5522847 92.4477153,18 93,18 C93.5522847,18 94,17.5522847 94,17 C94,16.4477153 93.5522847,16 93,16 Z M106,16 L106,18 L114,18 L114,16 L106,16 Z M117,16 C116.447715,16 116,16.4477153 116,17 C116,17.5522847 116.447715,18 117,18 C117.552285,18 118,17.5522847 118,17 C118,16.4477153 117.552285,16 117,16 Z M121,16 C120.447715,16 120,16.4477153 120,17 C120,17.5522847 120.447715,18 121,18 C121.552285,18 122,17.5522847 122,17 C122,16.4477153 121.552285,16 121,16 Z M100,18 C99.4477153,18 99,18.4477153 99,19 C99,19.5522847 99.4477153,20 100,20 C100.552285,20 101,19.5522847 101,19 C101,18.4477153 100.552285,18 100,18 Z M78,22 L78,24 L86,24 L86,22 L78,22 Z M93,22 C92.4477153,22 92,22.4477153 92,23 C92,23.5522847 92.4477153,24 93,24 C93.5522847,24 94,23.5522847 94,23 C94,22.4477153 93.5522847,22 93,22 Z M100,22 C99.4477153,22 99,22.4477153 99,23 C99,23.5522847 99.4477153,24 100,24 C100.552285,24 101,23.5522847 101,23 C101,22.4477153 100.552285,22 100,22 Z M106,22 L106,24 L114,24 L114,22 L106,22 Z M121,22 C120.447715,22 120,22.4477153 120,23 C120,23.5522847 120.447715,24 121,24 C121.552285,24 122,23.5522847 122,23 C122,22.4477153 121.552285,22 121,22 Z M100,26 C99.4477153,26 99,26.4477153 99,27 C99,27.5522847 99.4477153,28 100,28 C100.552285,28 101,27.5522847 101,27 C101,26.4477153 100.552285,26 100,26 Z M78,28 L78,30 L86,30 L86,28 L78,28 Z M89,28 C88.4477153,28 88,28.4477153 88,29 C88,29.5522847 88.4477153,30 89,30 C89.5522847,30 90,29.5522847 90,29 C90,28.4477153 89.5522847,28 89,28 Z M93,28 C92.4477153,28 92,28.4477153 92,29 C92,29.5522847 92.4477153,30 93,30 C93.5522847,30 94,29.5522847 94,29 C94,28.4477153 93.5522847,28 93,28 Z M100,30 C99.4477153,30 99,30.4477153 99,31 C99,31.5522847 99.4477153,32 100,32 C100.552285,32 101,31.5522847 101,31 C101,30.4477153 100.552285,30 100,30 Z M78,34 L78,36 L86,36 L86,34 L78,34 Z M93,34 C92.4477153,34 92,34.4477153 92,35 C92,35.5522847 92.4477153,36 93,36 C93.5522847,36 94,35.5522847 94,35 C94,34.4477153 93.5522847,34 93,34 Z M100,34 C99.4477153,34 99,34.4477153 99,35 C99,35.5522847 99.4477153,36 100,36 C100.552285,36 101,35.5522847 101,35 C101,34.4477153 100.552285,34 100,34 Z M100,38 C99.4477153,38 99,38.4477153 99,39 C99,39.5522847 99.4477153,40 100,40 C100.552285,40 101,39.5522847 101,39 C101,38.4477153 100.552285,38 100,38 Z M100,42 C99.4477153,42 99,42.4477153 99,43 C99,43.5522847 99.4477153,44 100,44 C100.552285,44 101,43.5522847 101,43 C101,42.4477153 100.552285,42 100,42 Z" id="icon-book"></path>
                    </g>
                </g>
            </g>
        </g>
    </g>
  </SvgIcon>
));

export default BookIcon;
