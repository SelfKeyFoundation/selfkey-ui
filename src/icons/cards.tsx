import * as React from 'react';
import { WithStyles, withStyles, createStyles } from '@material-ui/core';
import { SvgIcon } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';

const styles = createStyles({
    root: {
        fill: '#00C0D9',
        transition: 'all 0.2s ease-out',
    }
});

export const CardsIcon = withStyles(styles)((props: SvgIconProps & WithStyles<typeof styles>) => (
    <SvgIcon style={{ width: props.width || '44px', height: props.height || '34px' }} {...props} viewBox={props.viewBox || "0 0 44 34"}>
        <g id="🌐-Web-Marketplace" stroke="none" strokeWidth="1" fill="inherit" fillRule="evenodd">
            <g id="5000---Web-Marketplace" transform="translate(-260.000000, -1163.000000)" fill="inherit" fillRule="nonzero">
                <g id="Makretplaces-Boxes" transform="translate(230.000000, 426.000000)">
                    <g id="Exchange-Box-4" transform="translate(0.000000, 712.000000)">
                        <g id="icon-marketplace-cards" transform="translate(30.000000, 25.000000)">
                            <path d="M3.14285714,0 C1.41428571,0 0,1.41428571 0,3.14285714 L0,23.2571429 C0,24.9857143 1.41428571,26.4 3.14285714,26.4 L6.28571429,26.4 L6.28571429,30.8 C6.28571429,32.5285714 7.7,33.9428571 9.42857143,33.9428571 L40.8571429,33.9428571 C42.5857143,33.9428571 44,32.5285714 44,30.8 L44,11.9428571 C44,10.2142857 42.5857143,8.8 40.8571429,8.8 L38.9714286,8.8 L38.9714286,3.14285714 C38.9714286,1.41428571 37.5571429,0 35.8285714,0 L3.14285714,0 Z M3.14285714,1.25714286 L35.8285714,1.25714286 C36.8770091,1.25714286 37.7142857,2.09441949 37.7142857,3.14285714 L37.7142857,5.02857143 L1.25714286,5.02857143 L1.25714286,3.14285714 C1.25714286,2.09441949 2.09441949,1.25714286 3.14285714,1.25714286 Z M1.25714286,7.54285714 L37.7142857,7.54285714 L37.7142857,23.2571429 C37.7142857,24.3055805 36.8770091,25.1428571 35.8285714,25.1428571 L3.14285714,25.1428571 C2.09441949,25.1428571 1.25714286,24.3055805 1.25714286,23.2571429 L1.25714286,7.54285714 Z M38.9714286,10.0571429 L40.8571429,10.0571429 C41.9055805,10.0571429 42.7428571,10.8944195 42.7428571,11.9428571 L42.7428571,13.8285714 L38.9714286,13.8285714 L38.9714286,10.0571429 Z M3.77142857,11.9428571 C3.42522337,11.9428571 3.14285714,12.2252234 3.14285714,12.5714286 C3.14285714,12.9176338 3.42522337,13.2 3.77142857,13.2 C4.11763377,13.2 4.4,12.9176338 4.4,12.5714286 C4.4,12.2252234 4.11763377,11.9428571 3.77142857,11.9428571 Z M6.28571429,11.9428571 C5.93950909,11.9428571 5.65714286,12.2252234 5.65714286,12.5714286 C5.65714286,12.9176338 5.93950909,13.2 6.28571429,13.2 C6.63191949,13.2 6.91428571,12.9176338 6.91428571,12.5714286 C6.91428571,12.2252234 6.63191949,11.9428571 6.28571429,11.9428571 Z M8.8,11.9428571 C8.4537948,11.9428571 8.17142857,12.2252234 8.17142857,12.5714286 C8.17142857,12.9176338 8.4537948,13.2 8.8,13.2 C9.1462052,13.2 9.42857143,12.9176338 9.42857143,12.5714286 C9.42857143,12.2252234 9.1462052,11.9428571 8.8,11.9428571 Z M11.3142857,11.9428571 C10.9680805,11.9428571 10.6857143,12.2252234 10.6857143,12.5714286 C10.6857143,12.9176338 10.9680805,13.2 11.3142857,13.2 C11.6604909,13.2 11.9428571,12.9176338 11.9428571,12.5714286 C11.9428571,12.2252234 11.6604909,11.9428571 11.3142857,11.9428571 Z M13.8285714,11.9428571 C13.4823662,11.9428571 13.2,12.2252234 13.2,12.5714286 C13.2,12.9176338 13.4823662,13.2 13.8285714,13.2 C14.1747766,13.2 14.4571429,12.9176338 14.4571429,12.5714286 C14.4571429,12.2252234 14.1747766,11.9428571 13.8285714,11.9428571 Z M16.3428571,11.9428571 C15.9966519,11.9428571 15.7142857,12.2252234 15.7142857,12.5714286 C15.7142857,12.9176338 15.9966519,13.2 16.3428571,13.2 C16.6890623,13.2 16.9714286,12.9176338 16.9714286,12.5714286 C16.9714286,12.2252234 16.6890623,11.9428571 16.3428571,11.9428571 Z M18.8571429,11.9428571 C18.5109377,11.9428571 18.2285714,12.2252234 18.2285714,12.5714286 C18.2285714,12.9176338 18.5109377,13.2 18.8571429,13.2 C19.2033481,13.2 19.4857143,12.9176338 19.4857143,12.5714286 C19.4857143,12.2252234 19.2033481,11.9428571 18.8571429,11.9428571 Z M38.9714286,16.3428571 L42.7428571,16.3428571 L42.7428571,30.8 C42.7428571,31.8484377 41.9055805,32.6857143 40.8571429,32.6857143 L9.42857143,32.6857143 C8.38013377,32.6857143 7.54285714,31.8484377 7.54285714,30.8 L7.54285714,26.4 L35.8285714,26.4 C37.5571429,26.4 38.9714286,24.9857143 38.9714286,23.2571429 L38.9714286,16.3428571 Z M3.77142857,22 C3.42522337,22 3.14285714,22.2823662 3.14285714,22.6285714 C3.14285714,22.9747766 3.42522337,23.2571429 3.77142857,23.2571429 C4.11763377,23.2571429 4.4,22.9747766 4.4,22.6285714 C4.4,22.2823662 4.11763377,22 3.77142857,22 Z M6.28571429,22 C5.93950909,22 5.65714286,22.2823662 5.65714286,22.6285714 C5.65714286,22.9747766 5.93950909,23.2571429 6.28571429,23.2571429 C6.63191949,23.2571429 6.91428571,22.9747766 6.91428571,22.6285714 C6.91428571,22.2823662 6.63191949,22 6.28571429,22 Z M8.8,22 C8.4537948,22 8.17142857,22.2823662 8.17142857,22.6285714 C8.17142857,22.9747766 8.4537948,23.2571429 8.8,23.2571429 C9.1462052,23.2571429 9.42857143,22.9747766 9.42857143,22.6285714 C9.42857143,22.2823662 9.1462052,22 8.8,22 Z M11.3142857,22 C10.9680805,22 10.6857143,22.2823662 10.6857143,22.6285714 C10.6857143,22.9747766 10.9680805,23.2571429 11.3142857,23.2571429 C11.6604909,23.2571429 11.9428571,22.9747766 11.9428571,22.6285714 C11.9428571,22.2823662 11.6604909,22 11.3142857,22 Z M13.8285714,22 C13.4823662,22 13.2,22.2823662 13.2,22.6285714 C13.2,22.9747766 13.4823662,23.2571429 13.8285714,23.2571429 C14.1747766,23.2571429 14.4571429,22.9747766 14.4571429,22.6285714 C14.4571429,22.2823662 14.1747766,22 13.8285714,22 Z M16.3428571,22 C15.9966519,22 15.7142857,22.2823662 15.7142857,22.6285714 C15.7142857,22.9747766 15.9966519,23.2571429 16.3428571,23.2571429 C16.6890623,23.2571429 16.9714286,22.9747766 16.9714286,22.6285714 C16.9714286,22.2823662 16.6890623,22 16.3428571,22 Z M18.8571429,22 C18.5109377,22 18.2285714,22.2823662 18.2285714,22.6285714 C18.2285714,22.9747766 18.5109377,23.2571429 18.8571429,23.2571429 C19.2033481,23.2571429 19.4857143,22.9747766 19.4857143,22.6285714 C19.4857143,22.2823662 19.2033481,22 18.8571429,22 Z M21.3714286,22 C21.0252234,22 20.7428571,22.2823662 20.7428571,22.6285714 C20.7428571,22.9747766 21.0252234,23.2571429 21.3714286,23.2571429 C21.7176338,23.2571429 22,22.9747766 22,22.6285714 C22,22.2823662 21.7176338,22 21.3714286,22 Z M23.8857143,22 C23.5395091,22 23.2571429,22.2823662 23.2571429,22.6285714 C23.2571429,22.9747766 23.5395091,23.2571429 23.8857143,23.2571429 C24.2319195,23.2571429 24.5142857,22.9747766 24.5142857,22.6285714 C24.5142857,22.2823662 24.2319195,22 23.8857143,22 Z M26.4,22 C26.0537948,22 25.7714286,22.2823662 25.7714286,22.6285714 C25.7714286,22.9747766 26.0537948,23.2571429 26.4,23.2571429 C26.7462052,23.2571429 27.0285714,22.9747766 27.0285714,22.6285714 C27.0285714,22.2823662 26.7462052,22 26.4,22 Z M28.9142857,22 C28.5680805,22 28.2857143,22.2823662 28.2857143,22.6285714 C28.2857143,22.9747766 28.5680805,23.2571429 28.9142857,23.2571429 C29.2604909,23.2571429 29.5428571,22.9747766 29.5428571,22.6285714 C29.5428571,22.2823662 29.2604909,22 28.9142857,22 Z M31.4285714,22 C31.0823662,22 30.8,22.2823662 30.8,22.6285714 C30.8,22.9747766 31.0823662,23.2571429 31.4285714,23.2571429 C31.7747766,23.2571429 32.0571429,22.9747766 32.0571429,22.6285714 C32.0571429,22.2823662 31.7747766,22 31.4285714,22 Z M33.9428571,22 C33.5966519,22 33.3142857,22.2823662 33.3142857,22.6285714 C33.3142857,22.9747766 33.5966519,23.2571429 33.9428571,23.2571429 C34.2890623,23.2571429 34.5714286,22.9747766 34.5714286,22.6285714 C34.5714286,22.2823662 34.2890623,22 33.9428571,22 Z M10.0571429,29.5428571 C9.71093766,29.5428571 9.42857143,29.8252234 9.42857143,30.1714286 C9.42857143,30.5176338 9.71093766,30.8 10.0571429,30.8 C10.4033481,30.8 10.6857143,30.5176338 10.6857143,30.1714286 C10.6857143,29.8252234 10.4033481,29.5428571 10.0571429,29.5428571 Z M12.5714286,29.5428571 C12.2252234,29.5428571 11.9428571,29.8252234 11.9428571,30.1714286 C11.9428571,30.5176338 12.2252234,30.8 12.5714286,30.8 C12.9176338,30.8 13.2,30.5176338 13.2,30.1714286 C13.2,29.8252234 12.9176338,29.5428571 12.5714286,29.5428571 Z M15.0857143,29.5428571 C14.7395091,29.5428571 14.4571429,29.8252234 14.4571429,30.1714286 C14.4571429,30.5176338 14.7395091,30.8 15.0857143,30.8 C15.4319195,30.8 15.7142857,30.5176338 15.7142857,30.1714286 C15.7142857,29.8252234 15.4319195,29.5428571 15.0857143,29.5428571 Z M17.6,29.5428571 C17.2537948,29.5428571 16.9714286,29.8252234 16.9714286,30.1714286 C16.9714286,30.5176338 17.2537948,30.8 17.6,30.8 C17.9462052,30.8 18.2285714,30.5176338 18.2285714,30.1714286 C18.2285714,29.8252234 17.9462052,29.5428571 17.6,29.5428571 Z M20.1142857,29.5428571 C19.7680805,29.5428571 19.4857143,29.8252234 19.4857143,30.1714286 C19.4857143,30.5176338 19.7680805,30.8 20.1142857,30.8 C20.4604909,30.8 20.7428571,30.5176338 20.7428571,30.1714286 C20.7428571,29.8252234 20.4604909,29.5428571 20.1142857,29.5428571 Z M22.6285714,29.5428571 C22.2823662,29.5428571 22,29.8252234 22,30.1714286 C22,30.5176338 22.2823662,30.8 22.6285714,30.8 C22.9747766,30.8 23.2571429,30.5176338 23.2571429,30.1714286 C23.2571429,29.8252234 22.9747766,29.5428571 22.6285714,29.5428571 Z M25.1428571,29.5428571 C24.7966519,29.5428571 24.5142857,29.8252234 24.5142857,30.1714286 C24.5142857,30.5176338 24.7966519,30.8 25.1428571,30.8 C25.4890623,30.8 25.7714286,30.5176338 25.7714286,30.1714286 C25.7714286,29.8252234 25.4890623,29.5428571 25.1428571,29.5428571 Z M27.6571429,29.5428571 C27.3109377,29.5428571 27.0285714,29.8252234 27.0285714,30.1714286 C27.0285714,30.5176338 27.3109377,30.8 27.6571429,30.8 C28.0033481,30.8 28.2857143,30.5176338 28.2857143,30.1714286 C28.2857143,29.8252234 28.0033481,29.5428571 27.6571429,29.5428571 Z M30.1714286,29.5428571 C29.8252234,29.5428571 29.5428571,29.8252234 29.5428571,30.1714286 C29.5428571,30.5176338 29.8252234,30.8 30.1714286,30.8 C30.5176338,30.8 30.8,30.5176338 30.8,30.1714286 C30.8,29.8252234 30.5176338,29.5428571 30.1714286,29.5428571 Z M32.6857143,29.5428571 C32.3395091,29.5428571 32.0571429,29.8252234 32.0571429,30.1714286 C32.0571429,30.5176338 32.3395091,30.8 32.6857143,30.8 C33.0319195,30.8 33.3142857,30.5176338 33.3142857,30.1714286 C33.3142857,29.8252234 33.0319195,29.5428571 32.6857143,29.5428571 Z M35.2,29.5428571 C34.8537948,29.5428571 34.5714286,29.8252234 34.5714286,30.1714286 C34.5714286,30.5176338 34.8537948,30.8 35.2,30.8 C35.5462052,30.8 35.8285714,30.5176338 35.8285714,30.1714286 C35.8285714,29.8252234 35.5462052,29.5428571 35.2,29.5428571 Z M37.7142857,29.5428571 C37.3680805,29.5428571 37.0857143,29.8252234 37.0857143,30.1714286 C37.0857143,30.5176338 37.3680805,30.8 37.7142857,30.8 C38.0604909,30.8 38.3428571,30.5176338 38.3428571,30.1714286 C38.3428571,29.8252234 38.0604909,29.5428571 37.7142857,29.5428571 Z M40.2285714,29.5428571 C39.8823662,29.5428571 39.6,29.8252234 39.6,30.1714286 C39.6,30.5176338 39.8823662,30.8 40.2285714,30.8 C40.5747766,30.8 40.8571429,30.5176338 40.8571429,30.1714286 C40.8571429,29.8252234 40.5747766,29.5428571 40.2285714,29.5428571 Z" id="Shape-cards"></path>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </SvgIcon>
));

export default CardsIcon;
