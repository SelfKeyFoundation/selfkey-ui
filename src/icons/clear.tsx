import * as React from 'react';
import { SvgIcon } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';

export const ClearIcon = ((props: SvgIconProps) => (
        <SvgIcon width="34px" height="20px" {...props} viewBox="0 0 34 20">
            <defs></defs>
            <g id="ðŸ› -Wallet-Setup" stroke="none" strokeWidth="1" fillRule="evenodd">
                <g id="118---Trezor---Enter-PIN" transform="translate(-810.000000, -640.000000)">
                    <g id="Modal1" transform="translate(323.000000, 100.000000)">
                        <g id="input" transform="translate(260.000000, 535.000000)">
                            <path d="M243.38,12.6433333 L241.023333,15 L243.38,17.3566667 L241.023333,19.715 L238.666667,17.3566667 L236.308333,19.715 L233.951667,17.3566667 L236.308333,15 L233.951667,12.6433333 L236.308333,10.2866667 L238.666667,12.6433333 L241.023333,10.2866667 L243.38,12.6433333 Z M245.888333,21.6666667 L230.333333,21.6666667 L230.333333,8.33333333 L245.888333,8.33333333 L254.776667,15 L245.888333,21.6666667 Z M247,5 L227,5 L227,25 L247,25 L260.333333,15 L247,5 Z" id="icon-del-arrow" transform="translate(243.666667, 15.000000) scale(-1, 1) translate(-243.666667, -15.000000) "></path>
                        </g>
                    </g>
                </g>
            </g>
        </SvgIcon>
    )
);

export default ClearIcon;
