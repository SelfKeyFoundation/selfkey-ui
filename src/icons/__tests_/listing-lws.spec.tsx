import * as React from 'react';
import { create } from 'react-test-renderer';
import { ListingLWSIcon } from '../../icons/listing-lws';

describe('Listing LWS Icon', () => {
  it('should show Listing LWS Icon', () => {
    const component = create(
      <ListingLWSIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});