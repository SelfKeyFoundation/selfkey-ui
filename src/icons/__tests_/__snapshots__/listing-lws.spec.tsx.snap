// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listing LWS Icon should show Listing LWS Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root"
  focusable="false"
  style={
    Object {
      "height": "21px",
      "width": "10px",
    }
  }
  viewBox="0 0 10 21"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸŒˆ-UIelinesxx"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="#00C0D9"
      id="6uidexts-2"
      transform="translate(-1053.000000, -825.000000)"
    >
      <g
        id="Badgesx"
        transform="translate(1047.000000, 679.000000)"
      >
        <path
          d="M14.3706239,146 C14.8997523,146.002842 15.3273555,146.433256 15.327915,146.963584 L15.3248203,146.960999 C15.3242608,147.490811 14.8966576,147.921225 14.3675292,147.924066 L10.8070872,147.924066 L10.8070872,149.999081 L13.4334483,149.999081 C13.6878851,149.999629 13.9316835,150.101457 14.1112106,150.282164 C14.2907376,150.462871 14.3912872,150.707655 14.3907394,150.962665 C14.3915949,151.493494 13.9630837,151.924591 13.4334483,151.925732 L10.8070872,151.925732 L10.8070872,159.312865 C10.8086345,159.346984 10.8086345,159.381102 10.8070872,159.41522 C12.6649334,159.907351 13.8728945,161.708385 13.6330559,163.626248 C13.3927015,165.544111 11.7783044,166.986903 9.85753284,166.999914 C7.93624547,167.01275 6.3027644,165.592187 6.03713674,163.677426 C5.77150908,161.762664 6.95522842,159.946122 8.8068852,159.428661 C8.80241209,159.391431 8.8000014,159.353981 8.7996643,159.316484 L8.7996643,147.010626 C8.79897952,146.743098 8.90443005,146.486269 9.0927889,146.29671 C9.2811478,146.107151 9.53696431,146.000411 9.80389154,146 C9.88796395,146 9.9715206,146.011373 10.0530142,146.032567 C10.1319019,146.011264 10.2132154,146.000317 10.2949158,146 L14.3706239,146 Z M9.84566984,161.044112 C8.65496155,161.044112 7.68970176,162.017564 7.68970176,163.218379 C7.68970176,164.419194 8.65496155,165.392647 9.84566984,165.392647 C11.0363781,165.392647 12.0016379,164.419194 12.0016379,163.218379 C12.0016379,162.017564 11.0363781,161.044112 9.84566984,161.044112 Z"
          id="icon-listing-lws"
        />
      </g>
    </g>
  </g>
</svg>
`;
